app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddCyclewayPartSurface.kt

```diff
@@ -25,6 +25,7 @@ class AddCyclewayPartSurface : OsmFilterQuestType<SurfaceAnswer>() {
             and !note:cycleway:surface
           )
         )
+        and (access !~ private|no or (foot and foot !~ private|no))
```

Comment:
what about `access=no bicycle=yes`  taggings? (some RoW in England have such tagging from what I remember)

Though maybe excluding bicycle-only ways is fine

Reply:
```suggestion
        and (access !~ private|no or (foot and foot !~ private|no))
        and !bicycle or bicycle !~ private|no
```

Reply:
I think you need `or` at proper level, this would block asking on `access=yes foot=yes bicycle=private` and will not enable asking and `access=no bicycle=yes` ways

Reply:
I think it should be

```suggestion
        and (access !~ private|no or foot !~ private|no or bicycle !~ private|no)
```